package uk.ac.bbk.REx.annotators;

import static org.junit.Assert.*;

import java.util.ArrayList;
import java.util.List;

import org.apache.uima.UIMAFramework;
import org.apache.uima.analysis_engine.AnalysisEngine;
import org.apache.uima.analysis_engine.AnalysisEngineDescription;
import org.apache.uima.analysis_engine.AnalysisEngineProcessException;
import org.apache.uima.jcas.JCas;
import org.apache.uima.jcas.tcas.Annotation;
import org.apache.uima.resource.ResourceInitializationException;
import org.apache.uima.util.InvalidXMLException;
import org.apache.uima.util.XMLInputSource;
import org.junit.Before;
import org.junit.Test;

import uk.ac.bbk.REx.types.Section;

public class SectionAnnotatorTest 
{
	AnalysisEngine sectionAE;
	JCas cas;
	
	@Before
	public void initialise() throws ResourceInitializationException, InvalidXMLException
	{
		XMLInputSource sectionIn = new XMLInputSource(getClass().getResourceAsStream("/uk/ac/bbk/REx/desc/SectionAnnotator.xml"), null);
		AnalysisEngineDescription sectionDesc = UIMAFramework.getXMLParser().parseAnalysisEngineDescription(sectionIn);
		sectionAE = UIMAFramework.produceAnalysisEngine(sectionDesc);
		
		cas = sectionAE.newJCas();
	}
	
	@Test
	public void test() throws AnalysisEngineProcessException 
	{
		cas.setDocumentText("Deletion Strains Reveal Metabolic Roles for Key Elemental Sulfur-Responsive Proteins in Pyrococcus furiosus† Stephanie L. Bridger1, Sonya M. Clarkson1, Karen Stirrett2,‡, Megan B. DeBarry2, Gina L. Lipscomb2, Gerrit J. Schut1, Janet Westpheling2, Robert A. Scott1 and Michael W. W. Adams1,* + Author Affiliations  1Department of Biochemistry & Molecular Biology, University of Georgia, Athens, Georgia 30602 2Department of Genetics, University of Georgia, Athens, Georgia 30602   Next Section ABSTRACT  Transcriptional and enzymatic analyses of Pyrococcus furiosus previously indicated that three proteins play key roles in the metabolism of elemental sulfur (S0): a membrane-bound oxidoreductase complex (MBX), a cytoplasmic coenzyme A-dependent NADPH sulfur oxidoreductase (NSR), and sulfur-induced protein A (SipA). Deletion strains, referred to as MBX1, NSR1, and SIP1, respectively, have now been constructed by homologous recombination utilizing the uracil auxotrophic COM1 parent strain (ΔpyrF). The growth of all three mutants on maltose was comparable without S0, but in its presence, the growth of MBX1 was greatly impaired while the growth of NSR1 and SIP1 was largely unaffected. In the presence of S0, MBX1 produced little, if any, sulfide but much more acetate (per unit of protein) than the parent strain, demonstrating that MBX plays a critical role in S0 reduction and energy conservation. In contrast, comparable amounts of sulfide and acetate were produced by NSR1 and the parent strain, indicating that NSR is not essential for energy conservation during S0 reduction. Differences in transcriptional responses to S0 in NSR1 suggest that two sulfide dehydrogenase isoenzymes provide a compensatory NADPH-dependent S0 reduction system. Genes controlled by the S0-responsive regulator SurR were not as highly regulated in MBX1 and NSR1. SIP1 produced the same amount of acetate but more sulfide than the parent strain. That SipA is not essential for growth on S0 indicates that it is not required for detoxification of metal sulfides, as previously suggested. A model is proposed for S0 reduction by P. furiosus with roles for MBX and NSR in bioenergetics and for SipA in iron-sulfur metabolism.  Previous Section Next Section INTRODUCTION  Pyrococcus furiosus is a hyperthermophilic archaeon that grows optimally near 100°C (5). It utilizes carbohydrates for growth and produces acetate, CO2, and H2. When elemental sulfur (S0) is present, hydrogen sulfide (H2S) is produced instead of H2 (1, 5, 22). Transcriptional and biochemical analyses revealed a novel S0-reducing system found only in the Thermococcales (22) involving two key enzymes: a 13-gene cluster encoding a membrane-bound oxidoreductase (MBX) and a cytoplasmic coenzyme A (CoA)-dependent NADPH sulfur oxidoreductase (NSR), which is proposed to reduce S0 and produce sulfide intracellularly. MBX was predicted to act as a respiratory ferredoxin NADP oxidoreductase, generating NADPH for NSR and creating an electrochemical gradient to drive ATP synthesis, although this activity could not be verified experimentally (22). The gene cluster encoding MBX is highly similar to that which encodes the H2-evolving, energy-conserving, membrane-bound NiFe-hydrogenase (MBH), except that in MBX the homolog of the catalytic subunit responsible for H2 production in MBH (mbxL, PF1442) lacks two key residues necessary for coordinating a NiFe center (24).  A regulatory transcription factor, S0 response regulator SurR, is thought to regulate the expression of almost all of the primary S0 response genes in P. furiosus (22) and within 10 min of S0 addition causes the upregulation of genes involved in S0 metabolism and downregulation of those involved in H2 metabolism (12). The DNA-binding activity of SurR is modulated by a redox-dependent conformational change whereby SurR is unable to bind DNA in the presence of S0 (27). Accordingly, the expression of the genes encoding both MBX and NSR increases and the expression of the operon encoding MBH (as well as those encoding two soluble NiFe-hydrogenases, SHI and SHII) decreases as part of the primary S0 response (22). SurR also appears to downregulate the expression of an operon (PF1327-PF1328, sudAB [12, 22]) that encodes sulfide dehydrogenase I (SuDH I [6, 16]). This catalyzes both the NADPH-dependent reduction of S0 (6, 16) and the ferredoxin-dependent reduction of NADP (15, 16) in vitro; hence, its true function is not clear. P. furiosus also contains a homolog of SuDH I referred to as SuDH II (PF1910-PF1911, sudXY [6]), which is upregulated during growth on peptides (21).  The secondary response of P. furiosus to S0 addition occurs within 30 min and is independent of SurR. It includes genes involved in iron-sulfur cluster metabolism (22), as well as the most highly expressed gene in S0-grown cells encoding sulfur-induced protein A or SipA (23). SipA expression is regulated in an iron-dependent manner by sulfide, the product of S0 reduction (4), as well as by oxidative stress (25, 26). While the function of SipA is not known, it has been proposed to prevent the precipitation of toxic intracellular insoluble metal sulfides (7) by the controlled reaction of sulfide with assimilated iron and by iron release due to oxidative damage to iron-sulfur clusters (4).  To provide further insight into the biochemical and physiological roles of these three key S0-responsive proteins, MBX, NSR, and SipA, we have taken advantage of the recently developed genetic system for P. furiosus (13) to construct and characterize targeted gene deletions. Their effects on growth and on the expression of genes involved in the primary and secondary responses to S0 show that MBX plays a critical role in S0 reduction and energy conservation while the two SuDH isoenzymes appear to compensate for the NADPH-dependent S0 reduction system in the absence of NSR. The strain lacking SipA grows well with S0, consistent with a role in iron-sulfur cluster metabolism rather than sulfide detoxification.  Previous Section Next Section MATERIALS AND METHODS  Strains and growth conditions.The P. furiosus strains used or constructed in this study are listed in Table 1. All strains were grown in the presence or absence of S0 with maltose as the primary carbon source. The growth medium was the same as previously reported (1), except that yeast extract was added at 0.5 g/liter and uracil (20 μM) was added to the growth medium of all auxotrophic strains (COM1, NSR1, and MBX1). Growth experiments to determine the effects of S0 were carried out in biological triplicate in 100-ml serum bottles with 50 ml medium stirred (300 rpm) at 98°C, with S0 (Alfa Aesar, Ward Hill, MA) added to a final concentration of 2 g/liter. To obtain RNA for quantitative PCR (qPCR) analyses, cultures were grown in a 20-liter fermentor (1) and samples (2 liters each) were removed before and 30 min after the addition of S0 as previously described (22).  View this table: In this window In a new window TABLE 1. P. furiosus strains constructed and/or used in this study  Construction of gene deletions.A deletion of nsr (PF1186) was constructed with 3-kb flanking regions cloned sequentially into a plasmid, and this plasmid was then used as the template to amplify the nsr deletion construct with only 1-kb flanking regions and cloned into pGLW015 containing the PgdhpyrF cassette for prototrophic selection (13). Therefore, the final markerless deletion of nsr contains a remnant of the original plasmid multiple cloning site (GCGGCCGCATTTAAATACAAGTATAGCGGAAGATATCGGCCGGCC) as a scar. A deletion of mbxL (PF1442) was constructed by overlap PCR with 1-kb flanking regions and cloned into pGLW015 (13). A deletion of sipA (PF2025) was constructed with 1-kb flanking regions on either side of the PgdhpyrF cassette (13) using overlap PCR. Plasmid deletion constructs for nsr and mbxL were transformed into P. furiosus COM1 (ΔpyrF) selecting uracil prototrophy on solid defined medium and counterselected for loss of the plasmid using 5-fluoroorotic acid resistance as previously described (13). The PCR product containing a deletion of sipA was transformed directly selecting uracil prototrophy, resulting in marker replacement. The ΔpyrF allele in the COM1 strain was restored to the wild type by transforming COM1 with a PCR product containing the wild-type pyrF (PF1114) allele and ∼1-kb flanking regions. DNA was extracted from transformants as previously described (13) and screened for deletion by PCR amplification of the locus using primers outside the homologous flanking regions used to construct the deletions. Isolates containing the deletions were further colony purified by serial passage on solid medium. PCR products amplified from the target regions were sequenced.  Cell protein, H2S, and H2 analyses.The Bradford method (2) was routinely used to estimate total cell protein concentrations to monitor cell growth, with bovine serum albumin as the standard. Headspace and medium samples (500 μl each) were taken from cultures and transferred anaerobically into the double-vial system for H2S and H2 analyses as previously reported (22). H2S production was assayed by the methylene blue method (3), and abiotic sulfide production was subtracted from the experimental samples using controls lacking cells. H2 production was measured with a gas chromatograph (Shimadzu GC-8A).  Acetate measurements.Samples (1 ml) of P. furiosus cultures were centrifuged at 16,000 × g for 20 min to pellet cells, and the supernatant fraction was acidified with 0.1 M (final concentration) H2SO4. Acetate concentrations were determined using a Waters 2690 high-performance liquid chromatography separation module equipped with a photodiode array detector. Organic acids were separated on an Aminex HPX-87H column (Bio-Rad) at 23°C using 5 mM H2SO4 as the mobile phase at 0.6 ml/min. The specific acetate production of each strain was calculated based on a known acetate standard and divided by the total estimated cell protein (described above) at the endpoint of growth.  RNA isolation and qPCR analyses.Total RNA was extracted from P. furious cells using acid-phenol (23) and further purified by a second acid-phenol isolation, Turbo DNase (Ambion, Austin, TX) treatment (30 min, 37°C), and the Absolutely RNA cleanup kit (Agilent Technologies, Lexington, MA). cDNA was prepared using the AffinityScript qPCR cDNA Synthesis kit (Agilent). The genes pdo (PF0094), shIβ (PF0891), nsr (PF1186), sudB (PF1328), shIIβ (PF1329), mbhA (PF1423), mbxA (PF1453), sudY (PF1911), sipA (PF2025), and PF2051 were selected for study, and the constitutively expressed gene encoding the pyruvate ferredoxin oxidoreductase (POR) gamma subunit (PF0971) was selected as a control. qPCR experiments were carried out in technical triplicate using an Mx3000P instrument (Agilent) and the Brilliant SYBR green qPCR master mix (Agilent). The comparative cycle threshold method was used to analyze the resulting data, which are expressed as a ratio of gene expression change (n-fold).  Previous Section Next Section RESULTS  Construction and validation of P. furiosus deletion strains.Markerless deletions of nsr (NSR1) and mbxL (MBX1) and a marker replacement-containing deletion of sipA (SIP1) were constructed in the COM1 background strain (ΔpyrF). PCR and sequence analyses confirmed the gene deletions in all three strains (see Fig. S1 and S2 in the supplemental material), and qPCR products of the deleted genes were not detected. The absence of NSR and SipA in the appropriate strain was also confirmed by Western analysis (see Fig. S3 in the supplemental material).  Effect of S0 availability on the growth of deletion strains.The growth of the deletion strains and that of the control strains were compared after three consecutive transfers with standard inocula (1 × 107 cells/ml) using a maltose-containing medium with and without 2 g/liter S0. NSR1 and MBX1 were compared to COM1 (ΔpyrF) in the presence of 20 μM uracil, and SIP1 was compared to complemented COM1c (ΔpyrF::pyrF). The growth of MBX1 was similar to that of COM1 in the absence of S0, but MBX1 was significantly impaired in both growth rate and final cell yield in the presence of S0 (Fig. 1). In contrast, the growth of NSR1 and SIP1 was comparable to that of the control strains in both the presence and the absence of S0, with similar growth rates and final cell yields (Fig. 1 and 2).   View larger version: In this page In a new window Download as PowerPoint Slide FIG. 1. Effect of S0 availability on the growth of NSR1 and MBX1. Cultures were grown in 100-ml bottles stirred at 98°C with 5 g/liter maltose, 0.5 g/liter yeast extract, and 20 μM uracil. Cell growth was monitored by assaying total cell protein at each time point. (Top) Maltose only. (Bottom) Maltose plus 2 g/liter S0. Results obtained with COM1 (closed circles), NSR1 (open squares), and MBX1 (open triangles) are shown.   View larger version: In this page In a new window Download as PowerPoint Slide FIG. 2. Effect of S0 availability on the growth of SIP1. Cultures were grown in 100-ml bottles stirred at 98°C with 5 g/liter maltose and 0.5 g/liter yeast extract. Cell growth was monitored by assaying total cell protein at each time point. (Top) Maltose only. (Bottom) Maltose plus 2 g/liter S0. Results obtained with COM1c (closed squares) and SIP1 (open circles) are shown.  Production of H2S, H2, and acetate in deletion strains.Using the appropriate control strains for comparison, the amounts of H2S, H2, and acetate produced during the growth of the three deletion strains with S0 were determined. Like COM1, NSR1 produced H2S at a rate closely following that of cell growth (see Fig. S4 in the supplemental material). Similarly, H2S production in COM1c and SIP1 followed cell growth (see Fig. S5 in the supplemental material); however, the total H2S (per microgram of protein) was higher in SIP1 than in COM1c (see Fig. S6 in the supplemental material). In contrast, given the very low growth rate of MBX1, coupled with the high background of abiotic S0 reduction, it was not possible to determine if this strain was actively reducing S0. Interestingly, compared to COM1 and NSR1, which produce little, if any, H2 during growth on S0 (2.3 ± 2 nmol H2 per μg protein), MBX1 did produce appreciable amounts of H2 (25 ± 7 nmol H2 per μg protein). This is about 14% of that measured when the strains are grown in the absence of S0 (182 ± 6 nmol H2 per μg of protein). Specific production of acetate was calculated for each strain based on the concentration of acetate in the medium and total cell protein after 11 h of growth. Acetate production (per μg cell protein) was similar in COM1, COM1c, NSR1, and SIP1 (152 ± 11.3 μmol) but significantly higher in MBX1 (190 ± 2.7 μmol).  Effect of S0 addition on the growth and transcriptional responses of deletion strains.All strains were challenged by the addition of S0 near mid-log phase (Fig. 3). The growth of NSR1, SIP1, and the control strains was affected similarly (Fig. 3), with a brief stall immediately upon S0 addition, followed by restoration of the initial growth rate within 30 min. However, while the growth of MBX1 also stalled briefly upon S0 addition before resuming within 30 min, an additional growth effect was observed 1 h after S0 addition where the growth rate lagged for 1 h before returning to that observed in the absence of S0, but for only 2 h before reaching stationary phase (Fig. 3).   View larger version: In this page In a new window Download as PowerPoint Slide FIG. 3. Effect of S0 addition on the growth of NSR1, MBX1, and SIP1. Kinetic growth curves of deletion strains compared to those of control strains designated in Table 1. Cultures were grown in 100-ml bottles stirred at 98°C with 5 g/liter maltose and 0.5 g/liter yeast extract prior to the addition of 2 g/liter S0 (as indicated by the arrows). Cell growth was monitored by assaying total cell protein at each time point. (Top) Uracil (20 μM) was added to the growth of COM1 (closed circles), NSR1 (open squares), and MBX1 (open triangles). (Bottom) COM1c (closed squares) and SIP1 (open circles).  Cells of all strains were harvested just before S0 addition and 30 min after (Fig. 3), and the transcriptional responses of previously identified S0-responsive genes (22) and those potentially involved in S0 metabolism (6) were determined. These included genes whose expression is upregulated upon the addition of S0, a glutaredoxin-like protein disulfide oxidoreductase (PF0094) which does not catalyze S0 reduction (22) and a putative operon consisting of two potential regulators, PF2051 and PF2052 (22). As shown in Fig. 4, compared to COM1, most of the S0-responsive genes exhibited a much smaller change in expression in NSR1 and MBX1. Specifically, gene expression in NSR1 compared to that in COM1 (Fig. 4A) was lower for the S0-upregulated genes mbxA (3- versus 6-fold), PF2051 (2- versus 10-fold) pdo (4- versus 7-fold), and sipA (4- versus 11-fold) and for the S0-downregulated genes shIβ (17- versus 95-fold) and shIIβ (193- versus 478-fold). Similarly, the expression in MBX1, compared to that in COM1 (Fig. 4B), was lower for the S0-upregulated genes nsr (8- versus 14-fold), PF2051 (8- versus 10-fold), pdo (3- versus 7-fold), and sipA (<2- versus 11-fold) and for the S0-downregulated genes shIβ (38- versus 95-fold) and shIIβ (80- versus 478-fold). In contrast, the degree of S0 downregulation of mbhA was seemingly unaffected in NSR1 and MBX1 compared to that in COM1 (13-, 12-, and 13-fold, respectively). A total of four genes in the MBX operon were examined in the MBX1 strain: mbxA (the first gene in the MBX operon, PF1453), mbxK (upstream of the deleted gene, PF1443), mbxL (deleted gene, PF1442), and mbxN (downstream of the deleted gene, PF1441). An increase in the expression of the genes mbxA, mbxK, and mbxN was observed 30 min after S0 addition (11-, 17-, and 10-fold, respectively), while no mbxL gene product was detected.   View larger version: In this page In a new window Download as PowerPoint Slide FIG. 4. Quantitative reverse transcription-PCR of select S0 response genes. Total RNA was prepared from COM1, NSR1, and MBX1 cells harvested before and 30 min after S0 (2 g/liter) addition. For gene clusters of interest, the first gene in the operon was selected for analysis. The constitutively expressed gene PF0971 (por gamma subunit) was used as a control. Shown is the ratio of change in gene expression within 30 min of S0 addition in deletion strains (open bars) to that in the appropriate control strains (closed bars). (A) COM1 (closed bars) and NSR1 (open bars). (B) COM1 (closed bars) and MBX1 (open bars). A single asterisk indicates qPCR confirmation of the deleted gene product. Double asterisks indicate that an increase in mbxA (PF1453) gene expression was observed in MBX1; however, no gene product was observed for the deleted L subunit (PF1442). v., versus.  An interesting expression pattern was observed for the genes encoding SuDH I (sudB) and SuDH II (sudY) following S0 addition in NSR1 and MBX1 (Fig. 4). In NSR1, the expression of sudB upon S0 addition decreased to a lesser extent (5-fold less) than in the parent strain while the expression of sudY was 7-fold higher than in the parent strain. Similarly, in MBX1, the expression of sudY was 2-fold higher than in the parent strain and the expression of sudB actually increased ∼2-fold following S0 addition, compared to an almost order-of-magnitude decrease in the parent strain (a difference in expression of ∼10-fold between the two strains).  For SIP1, only two of the primary S0 response genes were differentially expressed 30 min after S0 addition compared to that in COM1c. These were pdo, whose expression increased 7-fold in SIP1, compared to 13-fold in the control strain, and shIIβ, whose expression decreased 160-fold in SIP1, compared to 460-fold in the control strain (see Fig. S7 in the supplemental material).  Previous Section Next Section DISCUSSION  MBX1 exhibited a well-defined phenotype during batch growth in the presence of S0, with a dramatically lower growth rate and cell yield (Fig. 1), and produced little, if any, sulfide, suggesting that the pathway for the disposal of reductant from glycolysis via sulfide production is blocked and that MBX plays an obligatory role in mediating electron flow to S0 (Fig. 5). The decrease in cell yield observed for MBX1 can be attributed in part to a decrease in respiratory ATP production. In the absence of S0, the MBH respiratory system is proposed to account for an additional 1.2 mol of ATP per mol of glucose oxidized to acetate and CO2 (20). Therefore, it would be expected that if the equivalent ion-pumping mechanism were impaired in the MBX1 strain, it would exhibit a reduced cell yield due to the loss of ATP synthesis in respiration and be able to generate ATP only via substrate level phosphorylation, for a total of 2 mol ATP rather than 3.2 mol ATP per mol glucose. Support for this hypothesis was obtained by the finding that MBX1 produced 20% more acetate (per unit of protein) than the parent strain or NSR1 during growth on S0, indicating that MBX1 must oxidize more glucose to acetate to yield the same amount of ATP for cell growth. When challenged in exponential growth by the addition of S0 (Fig. 3), MBX1 initially responded in a manner similar to that of the parent strain but then underwent a 1-h lag phase before cells resumed their previous growth rate for an additional 2 h before reaching stationary phase, but at a lower final cell density than the parent strain. This lag in growth can be explained by the inability of MBX1 to generate a proton gradient for ATP synthesis in the presence of S0, and the resumption of the growth rate may be due to increased glycolytic flux.   View larger version: In this page In a new window Download as PowerPoint Slide FIG. 5. Proposed physiological roles of S0 reduction in P. furiosus: bioenergetics and FeS metabolism. MBX, membrane-bound oxidoreductase complex; NSR, NADPH sulfur oxidoreductase; SipA, sulfur-induced protein A; SurR, S0 response regulator; SuDH I and II, sulfide dehydrogenases I and II; Fd, ferredoxin; PS, polysulfide; H2S, hydrogen sulfide; CoASH, coenzyme A; NSR1, deletion strain lacking NSR. The precise mechanisms by which PS interacts with SurR and SipA involvement in FeS metabolism are not clear.  In addition to producing more acetate than the COM1 and NSR1 strains in the presence of S0, MBX1 also produced about 10-fold more H2 (per unit of protein). The gene expression results (Fig. 4) also indicate that the soluble hydrogenases, SHI and SHII, are significantly less downregulated in MBX1 than in the parent strain and that expression of H2-producing MBH is seemingly unaffected within 30 min after S0 addition. Similar results were reported for an MBX-deficient mutant (MXD1) in the related hyperthermophilic archaeon Thermococcus kodakarensis (9). In the presence of S0, the MXD1 strain produced 4 times as much H2 as the wild-type strain and an increase in the expression of the soluble and membrane-bound hydrogenases was observed (9). Hence, P. furiosus MBX1 also appears to retain some ability to dispose of excess reductant as H2, as well as increase the glycolytic flux to compensate for decreased ATP production per unit of glucose oxidized. MBX is proposed to oxidize ferredoxin and reduce NADP where the NADPH is used by NSR to reduce S0 (Fig. 5). However, for unknown reasons, in in vitro assays, the membranes of S0-grown P. furiosus cells do not catalyze the ferredoxin-dependent reduction of NADP or, as discussed below, catalyze the direct ferredoxin-dependent reduction of S0 (22).  NSR is the only S0-reducing enzyme detected in extracts of S0-grown P. furiosus cells that was not present in cells grown without S0, and its upregulation is part of the SurR-mediated primary response (22). However, NSR1 did not exhibit an obvious phenotype (Fig. 1) and produced sulfide (see Fig. S4 in the supplemental material) and acetate in amounts comparable to those produced by the parent strain, clearly demonstrating that NSR is not essential for growth with S0. This poses the fundamental question of what reduces S0 in the absence of NSR. Given the degree of homology between MBX and the ferredoxin-dependent H2-evolving MBH complex, which can be demonstrated in vitro (20), MBX is an obvious candidate to catalyze ferredoxin-dependent S0 reduction, but cell membranes do not catalyze this reaction in vitro. On the other hand, NADPH-dependent S0 reduction by SuDH I and SuDH II may be a mechanism to compensate for the reduction of S0 in the absence of NSR. The expression of SuDH I in the presence of S0 does not decrease as much as in the parent strain (4- versus 9-fold, respectively), and the expression of SuDH II increases by an order of magnitude (10-fold) in NSR1 (Fig. 4). While SuDH II has not been characterized and the specific activity of SuDH I is much lower than that of NSR in NADPH-dependent S0 reduction (7 versus 100 μmol sulfide produced/min/mg) (16, 22), the compensatory expression of the two SuDH enzymes could be partially responsible for S0 reduction in NSR1. It is also possible that S0 reduction in NSR1 is catalyzed inadvertently by enzymes that do not normally function to reduce S0 but do so in vitro, including SHI and SHII (17) and POR (G. J. Schut, unpublished data). These enzymes, in combination with SuDH I and SuDH II, may be able to generate amounts of sulfide in NSR1 comparable to those in the parent strain.  It has been shown that in the absence of S0, SurR activates the transcription of genes necessary for H2 metabolism and represses that of those required for growth with S0 (27). Addition of S0 leads to SurR oxidation, possibly mediated by either colloidal S0 or polysulfide, such that it is unable to bind DNA (27). Since the degree of the SurR-mediated gene regulation was up to 80% lower in the MBX1 and NSR1 deletion strains than in the parent strain (Fig. 4), there may be limited amounts of the agent that oxidizes SurR if MBX and NSR are not present. We postulate that NSR contributes to the pool of sulfur species responsible for SurR oxidation and that this NSR activity is dependent on the activity of MBX. However, NSR (and MBX) cannot provide the only source of oxidant for SurR, given that the growth of NSR1 is indistinguishable from that of the parent strain in the presence or absence of S0 (Fig. 1 and 3). The muted transcriptional responses in MBX1 and NSR1 also suggest that the product of the NSR reaction plays a role in mediating the increase in the expression of sipA during the secondary response to S0. It would be logical if one of the S0-reducing, SurR-regulated enzymes, namely, NSR, not only generated a product (presumably sulfide) that was utilized by SipA but also functioned in controlling its expression.  SipA is unlikely to play an essential role in the detoxification of metal sulfides, as previously proposed (4), as one would predict greatly impaired growth of the SIP1 mutant in the presence of S0, which was not observed (Fig. 2). In addition to upregulation of sipA expression, the secondary response to S0 includes the upregulation of genes involved in iron transport (feoB) and iron-sulfur cluster biosynthesis (sufBD) and of iron-sulfur-cluster-containing enzymes (glutamate synthase [18] and 3-isopropylmalate dehydratase [8]), indicating an increased need for iron-sulfur metabolism under S0-reducing conditions and that SipA may be involved in iron-sulfur metabolism. Since some archaea use sulfide, rather than cysteine, as a sulfur source for iron-sulfur cluster biosynthesis (14), NSR might function to enable S0 to be used, via sulfide, for iron-sulfur cluster synthesis by SipA. This is supported by the increased amount of sulfide produced by SIP1, suggesting that in the absence of SipA, sulfide is no longer incorporated into iron-sulfur clusters. If SipA does synthesize iron-sulfur clusters, a mutant lacking functional SufBD, the only recognized cluster-biosynthetic scaffold in P. furiosus, may grow only in the presence of S0, and such a study is currently in progress. It is unclear why the absence of SipA should influence the expression of the SHII genes in response to S0 but not that of the SHI genes (see Fig. S7 in the supplemental material).  While this paper was in revision, strains of T. kodakarensis with deletions of homologs of nsr and mbx, and also supposedly of sipA, were reported (19). The phenotypes of the nsr (TS1109) and mbx (TS1105) mutants are similar to those of their P. furiosus counterparts, although we did not observe the higher concentrations of H2 that were produced by the nsr mutant. The other T. kodakarensis strain is not comparable to P. furiosus SIP1 (19) because the deleted genes (TK1260 and TK1261) are homologs of an operon (PF2051-PF2052) encoding putative transcriptional regulators. In any event, the results obtained with both organisms are consistent with a role for MBX in energy conservation during S0 reduction and in connecting reduced ferredoxin generated during glycolysis to S0 reduction by NSR via NADPH. NSR likely reduces colloidal sulfur (22), which exists as both short S0 chains and S8 (10, 11), making the end products of S0 reduction both polysulfide (Sx2−) and hydrogen sulfide (H2S), as shown in Fig. 5. The products generated by NSR, which are assumed to be dependent on the NADPH generated by MBX, appear to modulate, at least in part, SurR activity and SipA expression. We propose that polysulfide oxidizes SurR while sulfide induces SipA expression (Fig. 5). When MBX is absent, NADPH is not generated and NSR cannot produce either polysulfide or H2S. We further propose that in the absence of NSR, a combination of SuDH I and II, and perhaps SHI, SHII, and POR, reduces S0. However, the muted transcriptional response in NSR1 suggests that the product(s) of SuDH/SH/POR reduction does not modulate SurR activity or SipA regulation. The reason for this is unclear and is the subject of further investigation.  Previous Section Next Section ACKNOWLEDGMENTS  This research was supported by grants from the Chemical Sciences, Geosciences, and Biosciences Division (FG05-95ER20175) and the Office of Biological and Environmental Research (FG02-08ER64690) of the Office of Basic Energy Sciences, Office of Science, U.S. Department of Energy.  We thank Michael Thorgersen and Frank Jenney for many helpful discussions and Farris Poole for invaluable technical assistance.  Previous Section Next Section FOOTNOTES  Received 2 June 2011. Accepted 21 September 2011. ↵*Corresponding author. Mailing address: Department of Biochemistry and Molecular Biology, Life Sciences Bldg., University of Georgia, Athens, GA 30602-7229. Phone: (706) 542-2060. Fax: (706) 542-0229. E-mail: adams@bmb.uga.edu. ↵‡ Present address: Southeastern Community College, Whiteville, NC 28452. ↵† Supplemental material for this article may be found at http://jb.asm.org/. ↵▿ Published ahead of print on 30 September 2011. Previous Section   REFERENCES  1.↵ Adams M. W., et al. 2001. Key role for sulfur in peptide metabolism and in regulation of three hydrogenases in the hyperthermophilic archaeon Pyrococcus furiosus. J. Bacteriol. 183:716–724. Abstract/FREE Full Text 2.↵ Bradford M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 72:248–254. CrossRefMedline 3.↵ Chen J. S., Mortenson L. E.. 1977. Inhibition of methylene blue formation during determination of the acid-labile sulfide of iron-sulfur protein samples containing dithionite. Anal. Biochem. 79:157–165. CrossRefMedline 4.↵ Clarkson S. M., Newcomer E. C., Young E. G., Adams M. W.. 2010. The elemental sulfur-responsive protein (SipA) from the hyperthermophilic archaeon Pyrococcus furiosus is regulated by sulfide in an iron-dependent manner. J. Bacteriol. 192:5841–5843. Abstract/FREE Full Text 5.↵ Fiala G., S. K. O. 1986. Pyrococcus furiosus sp. nov. represents a novel genus of marine heterotrophic archaebacteria growing optimally at 100°C. Arch. Microbiol. 145:56–61. CrossRef 6.↵ Hagen W. R., et al. 2000. Novel structure and redox chemistry of the prosthetic groups of the iron-sulfur flavoprotein sulfide dehydrogenase from Pyrococcus furiosus; evidence for a [2Fe-2S] cluster with Asp(Cys)3 ligands. J. Biol. Inorg. Chem. 5:527–534. CrossRefMedline 7.↵ Hatton B., Rickard D.. 2008. Nucleic acids bind to nanoparticulate iron (II) monosulphide in aqueous solutions. Orig. Life Evol. Biosph. 38:257–270. CrossRefMedline 8.↵ Jang S., Imlay J. A.. 2007. Micromolar intracellular hydrogen peroxide disrupts metabolism by damaging iron-sulfur enzymes. J. Biol. Chem. 282:929–937. Abstract/FREE Full Text 9.↵ Kanai T., et al. 2011. Distinct physiological roles of the three [NiFe]-hydrogenase orthologs in the hyperthermophilic archaeon Thermococcus kodakarensis. J. Bacteriol. 193:3109–3116. Abstract/FREE Full Text 10.↵ Kleinjan W. E., de Keizer A., Janssen A. J.. 2005. Kinetics of the chemical oxidation of polysulfide anions in aqueous solution. Water Res. 39:4093–4100. CrossRefMedline 11.↵ Kleinjan W. E., de Keizer A., Janssen J. H.. 2003. Biologically produced sulfur. Top. Curr. Chem. 230:167–188. Search Google Scholar 12.↵ Lipscomb G. L., et al. 2009. SurR: a transcriptional activator and repressor controlling hydrogen and elemental sulphur metabolism in Pyrococcus furiosus. Mol. Microbiol. 71:332–349. CrossRefMedline 13.↵ Lipscomb G. L., et al. 2011. Natural competence in the hyperthermophilic archaeon Pyrococcus furiosus facilitates genetic manipulation: construction of multiple markerless deletions of genes encoding the two cytoplasmic hydrogenases. Appl. Environ. Microbiol. 77:2232–2238. Abstract/FREE Full Text 14.↵ Liu Y., Sieprawska-Lupa M., Whitman W. B., White R. H.. 2010. Cysteine is not the sulfur source for iron-sulfur cluster and methionine biosynthesis in the methanogenic archaeon Methanococcus maripaludis. J. Biol. Chem. 285:31923–31929. Abstract/FREE Full Text 15.↵ Ma K., Adams M. W.. 2001. Ferredoxin:NADP oxidoreductase from Pyrococcus furiosus. Methods Enzymol. 334:40–45. Medline 16.↵ Ma K., Adams M. W.. 1994. Sulfide dehydrogenase from the hyperthermophilic archaeon Pyrococcus furiosus: a new multifunctional enzyme involved in the reduction of elemental sulfur. J. Bacteriol. 176:6509–6517. Abstract/FREE Full Text 17.↵ Ma K., Schicho R. N., Kelly R. M., Adams M. W.. 1993. Hydrogenase of the hyperthermophile Pyrococcus furiosus is an elemental sulfur reductase or sulfhydrogenase: evidence for a sulfur-reducing hydrogenase ancestor. Proc. Natl. Acad. Sci. U. S. A. 90:5341–5344. Abstract/FREE Full Text 18.↵ Miller R. E., Stadtman E. R.. 1972. Glutamate synthase from Escherichia coli. An iron-sulfide flavoprotein. J. Biol. Chem. 247:7407–7419. Abstract/FREE Full Text 19.↵ Santangelo T. J., Cubonova L., Reeve J. N.. 2011. Deletion of alternative pathways for reductant recycling in Thermococcus kodakarensis increases hydrogen production. Mol. Microbiol. 81:897–911. CrossRefMedline 20.↵ Sapra R., Bagramyan K., Adams M. W.. 2003. A simple energy-conserving system: proton reduction coupled to proton translocation. Proc. Natl. Acad. Sci. U. S. A. 100:7545–7550. Abstract/FREE Full Text 21.↵ Schut G. J., Brehm S. D., Datta S., Adams M. W.. 2003. Whole-genome DNA microarray analysis of a hyperthermophile and an archaeon: Pyrococcus furiosus grown on carbohydrates or peptides. J. Bacteriol. 185:3935–3947. Abstract/FREE Full Text 22.↵ Schut G. J., Bridger S. L., Adams M. W.. 2007. Insights into the metabolism of elemental sulfur by the hyperthermophilic archaeon Pyrococcus furiosus: characterization of a coenzyme A-dependent NAD(P)H sulfur oxidoreductase. J. Bacteriol. 189:4431–4441. Abstract/FREE Full Text 23.↵ Schut G. J., Zhou J., Adams M. W.. 2001. DNA microarray analysis of the hyperthermophilic archaeon Pyrococcus furiosus: evidence for a new type of sulfur-reducing enzyme complex. J. Bacteriol. 183:7027–7036. Abstract/FREE Full Text 24.↵ Silva P. J., et al. 2000. Enzymes of hydrogen metabolism in Pyrococcus furiosus. Eur. J. Biochem. 267:6541–6551. Medline 25.↵ Strand K. R., et al. 2010. Oxidative stress protection and the repair response to hydrogen peroxide in the hyperthermophilic archaeon Pyrococcus furiosus and in related species. Arch. Microbiol. 192:447–459. CrossRefMedline 26.↵ Williams E., Lowe T. M., Savas J., DiRuggiero J.. 2007. Microarray analysis of the hyperthermophilic archaeon Pyrococcus furiosus exposed to gamma irradiation. Extremophiles 11:19–29. CrossRefMedline 27.↵ Yang H., et al. 2010. SurR regulates hydrogen production in Pyrococcus furiosus by a sulfur-dependent redox switch. Mol. Microbiol. 77:1111–1122. CrossRefMedline Copyright © 2011, American Society for Microbiology. All Rights Reserved.");
		sectionAE.process(cas);
		
		List<Integer> results = new ArrayList<Integer>();
		for(Annotation section : cas.getAnnotationIndex(Section.type))
		{
			results.add(section.getBegin());
		}
		
		assertEquals("[6059, 11093, 20047, 31161]", results.toString());
	}
}
